locals.tf

```diff
@@ -0,0 +1,10 @@
+resource "random_string" "suffix" {
+  length = 8
+  special = false
+}
+
+locals {
+  cluster_name = lower("jenkins-infra-doks-${random_string.suffix.result}")
+  doks_version = data.digitalocean_kubernetes_versions.k8s.latest_version
```

Comment:
This means that anyone running the master branch could update the cluster to the latest version. Just checking that’s what we want

Reply:
Yes. Not completly sure about this one, I propose we test it to see how it goes, and pin it if it's not pratical.